indico/modules/events/ical.py

```diff
@@ -110,24 +134,38 @@ def generate_event_component(event, user=None):
     return component
 
 
-def event_to_ical(event, user=None, scope=None, *, skip_access_check=False):
+def event_to_ical(
+        event: Event,
+        user: t.Optional[User] = None,
+        scope: t.Optional[str] = None,
+        *,
+        skip_access_check: bool = False,
+        method: t.Optional[str] = None):
     """Serialize an event into an ical.
 
     :param event: The event to serialize
     :param user: The user who needs to be able to access the events
     :param scope: If specified, use a more detailed timetable using the given scope
     :param skip_access_check: Do not perform access checks. Defaults to False.
+    :param method: METHOD field of the iCalendar object
     """
-    return events_to_ical([event], user, scope, skip_access_check=skip_access_check)
+    return events_to_ical([event], user, scope, skip_access_check=skip_access_check, method=method)
 
 
-def events_to_ical(events, user=None, scope=None, *, skip_access_check=False):
+def events_to_ical(
```

Comment:
I know it's a matter of preference, but what about using this indentation style for multiline function definitions?

```python
def events_to_ical(
    events: list[Event],
    user: t.Optional[User] = None,
    scope: t.Optional[str] = None,
    *,
    skip_access_check: bool = False,
    method: t.Optional[str] = None
):
    ...
```

I think that's actually the one we use most commonly in the codebase

Reply:
Indeed.